src/System/Hooks/usePrefetchRoute.tsx

```diff
@@ -63,6 +64,9 @@ export const usePrefetchRoute = (
           fetchPolicy: "store-or-network",
           networkCacheConfig: {
             force: false,
+            metadata: {
+              maxAge: serverCacheTTL,
```

Comment:
this is fine for now, but this might also undefined, which is maybe a 'tricky' thing to store as a root field in `metadata` - perhaps it should be nested under `prefetchInfo` or something.

We can improve that if we need to add more stuff.